src/MetaMorpho.sol

```diff
@@ -214,14 +214,12 @@ contract MetaMorpho is ERC4626, ERC20Permit, Ownable2Step, Multicall, IMetaMorph
     /// @inheritdoc IMetaMorphoBase
     function submitTimelock(uint256 newTimelock) external onlyOwner {
         if (newTimelock == timelock) revert ErrorsLib.AlreadySet();
+        if (pendingTimelock.validAt != 0) revert ErrorsLib.AlreadyPending();
```

Comment:
Before, you could have a pending value (decrease of timelock) that could have been deleted by another `submitTimelock` that set the new timelock to a new one.

Now you won't be able to do that, and you are forced to wait for the timelock to expire or revoke it and call it again.

Is this the intended new behavior?

Reply:
yes we enforce to revoke every pending values before submitting a new one.